v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -56,6 +56,16 @@ export const MovableLine = (props: {
     xSubAxesCount = layout.getAxisMultiScale('bottom')?.repetitions ?? 1,
     ySubAxesCount = layout.getAxisMultiScale('left')?.repetitions ?? 1
 
+  const equationContainerClass = useCallback(() => {
+    const classFromKey = model.setClassNameFromKey(lineKey)
```

Comment:
This has come up before, but this naming makes it seems like this method is setting a property on the model. It should be named something like `classNameFromKey()` instead.